Resources/doc/cookbook/recipe_knp_menu.rst

```diff
@@ -205,6 +205,29 @@ your admin services or simply remove menu items from the ``sonata_admin`` dashbo
                           label:        Blog
                         - sonata.news.admin.news
 
+Keeping menu group open
+-----------------------
+
+You can add the ``keep_open`` option to menu group to keep that group always
+open and ignore open/close effects:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+
+    sonata_admin:
+        dashboard:
+            groups:
+                news:
+                    keep_open:            true
+                    label:                ~
+                    label_catalogue:      ~
+                    items:
+                        - sonata.news.admin.post
+                        - route:        blog_home
+                          label:        Blog
+                        - sonata.news.admin.news
+
```

Comment:
please add the screenshot to the docs like this:

````
.. figure:: ../images/foo.png
   :align: center
   :alt: The navigation side bar with a group which uses "keep_open" option
```

Please make sure, the screenshot correspond to the docs (in this case a post and news entry should be shown)